src/lib.rs

```diff
@@ -471,19 +480,19 @@ pub struct MultiCursor<'a> {
 impl<'a> MultiCursor<'a> {
     /// Open a cursor in each of the Dbs passed. If you want multiple
     /// cursors for the same db, include it multiple times.
-    pub fn new(cursors: &[RefCell<Cursor<'a>>]) -> MultiCursor<'a> {
-        let mut ptrs: Vec<*const traildb_sys::tdb_cursor> = vec![];
-        for refcell in cursors.iter() {
-            let cursor = refcell.borrow();
-            let ptr: *const traildb_sys::tdb_cursor = cursor.obj;
-            ptrs.push(ptr);
-        }
+    pub fn new(cursors: &'a [Cursor<'a>]) -> MultiCursor<'a> {
+        use std::iter::FromIterator;
```

Comment:
Is there a reason you put the FromIterator inclusion into the method itself?

Reply:
The trait is only used for the method, so I thought it's better to include the trait inside the method. We could also use `cursors.into_iter().collect()` instead of `Self::from_iter(cursors)`. Does it sound better?